contrib/adapters/ffmpeg_burnins.py

```diff
@@ -0,0 +1,391 @@
+# OpenTimelineIO bundles ffmpeg_burnins, which is available under the MIT
+# License.  For details contrib/adapters.
```

Comment:
Can you move the text "OpenTimelineIO bundles ffmpeg_burnins, which is available under the MIT License.  For details contrib/adapters." to the bottom of the top-level LICENSE.txt file here:
https://github.com/PixarAnimationStudios/OpenTimelineIO/blob/master/LICENSE.txt

Reply:
and mention the exact filename of the ffmpeg_burnins.py file, to clarify which file(s) the MIT license applies to.

Reply:
The PIL issue can be resolved by editing the `.travis.yml` file. Look for `pip install` and add PIL to the list.

Reply:
I may have found a smarter way to do this. All examples of `ffmpeg` I looked at before were using `drawbox`, but just now found one that doesn't use it. Doing some tests now (if this works it will make the code cleaner too)

Reply:
You can make fun of me later for missing this (so much easier)

```
ffmpeg -loglevel panic -i BUSH0005.MOV -vf "drawtext=text='Bottom Right+50':x=1437.5:y=h-text_h-5:fontcolor=white@1.0:fontsize=48:fontfile=/System/Library/Fonts/Menlo.ttc:box=1:boxborderw=5:boxcolor=black" test.mov
```

Reply:
Nice!

Reply:
From a syntax perspective this is cleaner, however PIL getting the dimensions of the text is really useful and kind of needed. If you're using either of the right alignment parameters it helps to know how to offset the text from the right side (offsets appear to be calculated from the left side of the text). 

I think PIL will be needed, sorry for the complication.

Reply:
The alignment parameters are a convenience so that people don't have to bang their head against a keyboard getting the right x value (which can change on resolution). If you you the right alignments and give it an offset the tool can figure out what the actual X position of the text is.